yarn.lock

```diff
@@ -3841,7 +3841,21 @@
   integrity sha512-S9K+Q39BGOghyLHmR0Wdcmu1i1noSUk8HcvMj+3IaohZw02WFd99aPTQDHJeseXrXZP3CNovaSlePI0R11NcFg==
 
 "@cds/core@./dist/core":
-  version "5.2.0"
+  version "5.2.1"
+  dependencies:
+    "@types/resize-observer-browser" "^0.1.3"
+    lit-element "^2.3.1"
+    lit-html "^1.2.1"
+    ramda "^0.27.0"
+    tslib "^2.0.0"
+  optionalDependencies:
+    "@cds/city" "^1.1.0"
+    normalize.css "^8.0.1"
+
+"@cds/core@latest":
+  version "5.3.0"
```

Comment:
Something is weird in this yarn.lock file. 

The `@cds/core` here has the latest version (5.3.0) but there are other packages here pointing at `./dist` and 5.2.1.

That doesn't seem right.

Reply:
There are still packages that use `dist/<package>` to link packages - or at least on the next branch right now. 

So this is the reason for the change into the `yarn.lock` 

The change here is to make the adoption guide not depend on the pre-build packages but use only published ones. This is making moving the app super painless.

At the moment `apps/adoption` (where the app is located) is into the `nohoist` paths for Yarn, but it still affects the `yarn.lock` file when some of the packages change.